### ISSUE TITLE
Evaluate Model Performance

### ISSUE THREAD

**Issue by saadahmad22** [saadahmad22/machine-translator#1]
- Design the loss function
- Fine-tune as needed
